keras_cv/layers/preprocessing/vectorized_base_image_augmentation_layer_test.py

```diff
@@ -435,31 +435,26 @@ def test_augment_unbatched_all_data(self):
             "segmentation_masks": segmentation_masks,
         }
 
+        print(input)
```

Comment:
some prints escaped :)

Reply:
Welp that's embarassing -- removed, thanks 😄